src/main/java/de/dhbw/karlsruhe/domain/models/LeaderboardSaveEntry.java

```diff
@@ -0,0 +1,65 @@
+package de.dhbw.karlsruhe.domain.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class LeaderboardSaveEntry {
+
+    private UUID saveEntryID;
+    private int leaderboardTypeID;
+    private String username;
+    private int score;
+    private Date date;
```

Comment:
Wird nie benutzt, bzw. die Getter wird nie benutzt. Das date wird auch nicht in allen Konstruktoren gesetzt, das heißt es könnte zu Problemen kommen, sollte doch irgendwann der Getter benutzt werden.